programs/zstdcli.c

```diff
@@ -741,7 +741,7 @@ typedef enum { zom_compress, zom_decompress, zom_test, zom_bench, zom_train, zom
 # define MAXCLEVEL  ZSTD_maxCLevel()
 #endif
 
-int main(int const argCount, const char* argv[])
+int main(int CONST_ARGC argCount, const char* argv[])
```

Comment:
I'm puzzled by this proposed change. 
I get it that it's a workaround for elbrus compiler. And yes, it's correct.
I just wonder if it wouldn't be simpler to just remove the `const` statement.
This would avoid having to maintain yet another specificity.

Reply:
Yes, that was the first of my ideas--like why would you declare scalar local variable (or function parameter) as `const`? That may be useful only to capture some of programming mistakes (still there's nothing wrong in treating `argc` as not `const`, e.g. in loops like `for(int i = 0; argc; --argc, ++i) do_something_with(argv[i]);`), but still there's nothing to do with actual code flow. I then found commit a49417b and there was no explanation why `argCount` was changed to `const`; so I preferred not touching the part, reason of which I don't clearly understand. But if you're ok with that, I've just removed `const` at all.